### DISCUSSION TITLE
On OFF -> Night Mode how to track?

### DISCUSSION THREAD

**Discussion by clipse2004** [jseidl/magic-areas#167] [Q&A]
Hello, 

Good work for now! Really great your integration!

I have a question of understanding, though. The Night Mode should be activated as soon as it is too dark in a room. 
That works so far too, but how do I do it automatically? 
Does it have to be automated? I couldn’t find a template switch. 

Thank you

**Comment by jseidl** [comment]
Hi! Glad you liked it so far. You need to configure the night entity and night state for night mode auto-switching to work.

**Comment by jseidl** [comment]
Is this resolved?
